src/frontend/src/pages/WorkPackageDetailPage/WorkPackageEditContainer/WorkPackageFormDetails.tsx

```diff
@@ -66,7 +67,7 @@ const WorkPackageEditDetails: React.FC<Props> = ({
   return (
     <PageBlock title="Work Package Details">
```

Comment:
The main change we want for this ticket is removing the pageblock and making sure the website matches the mocks from there